AI-and-Analytics/Getting-Started-Samples/README.md

```diff
@@ -16,19 +16,18 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
 
 | Compoment      | Folder                                             | Description
```

Comment:
I have a request for changing this table structure. We need to map samples to Presets, so we can add a column for Preset on the left and then group samples by Preset.
We need this change because on AI Tools Selector page we provide a link to this README (Get Started samples folder README) instead of specific samples. So user may get lost when going here because the relation to Presets is not clear.